portfolio/src/main/java/com/google/sps/servlets/FunnyServlet.java

```diff
@@ -13,7 +13,7 @@
 /** Handles request sent to /funny */
 @WebServlet("/funny")
 public class FunnyServlet extends HttpServlet {
-  private ArrayList<String> jokeArray = new ArrayList<String>();
+  private static final ArrayList<String> jokeArray = new ArrayList<String>();
```

Comment:
We can talk about this in our 1:1 since it's hard to explain here but I would initialize this like you are here but assign it into a `java.util.List<String>`

Reply:
okay, thanks